src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -1574,15 +1576,21 @@ private boolean isRepositoryVisibleToTeam(GHOrganization org, GHRepository repo,
     // trusted source
     listener.getLogger().printf("Looking up details of %s...%n", getRepoOwner());
     List<Action> result = new ArrayList<>();
+    String apiUri = Util.fixEmptyAndTrim(getApiUri());
     StandardCredentials credentials =
-        Connector.lookupScanCredentials((Item) owner, getApiUri(), credentialsId);
-    GitHub hub = Connector.connect(getApiUri(), credentials);
+        Connector.lookupScanCredentials((Item) owner, apiUri, credentialsId);
+    GitHub hub = Connector.connect(apiUri, credentials);
+    boolean privateMode = apiUri == null ? false : determinePrivateMode(apiUri);
```

Comment:
@Vlatombe I missed one, sorry
```suggestion
    boolean privateMode = determinePrivateMode(apiUri);
```